connectors/services/access_control_sync_job_execution.py

```diff
@@ -31,71 +29,23 @@ def load_max_concurrent_access_control_syncs(config):
     )
 
 
-def load_max_concurrent_content_syncs(config):
-    max_concurrent_content_syncs = config.get("max_concurrent_content_syncs")
-
-    if max_concurrent_content_syncs is not None:
-        return max_concurrent_content_syncs
-
-    logger.warning(
-        "'max_concurrent_syncs' is deprecated. Use 'max_concurrent_content_syncs' in 'config.yml'."
-    )
-
-    # keep for backwards compatibility
-    return config.get("max_concurrent_syncs", DEFAULT_MAX_CONCURRENT_CONTENT_SYNCS)
-
-
-class JobExecutionService(BaseService):
-    name = "execute"
+class AccessControlSyncJobExecutionService(BaseService):
+    name = "execute_access_control_sync_job"
 
     def __init__(self, config):
         super().__init__(config)
         self.idling = self.service_config["idling"]
-        self.max_concurrent_content_syncs = load_max_concurrent_content_syncs(
-            self.service_config
-        )
+        self.source_list = config["sources"]
         self.max_concurrent_access_control_syncs = (
             load_max_concurrent_access_control_syncs(self.service_config)
         )
```

Comment:
`load_max_concurrent_access_control_syncs` can be inlined

Reply:
Did you mean
```python
self.max_concurrent_access_control_syncs = load_max_concurrent_access_control_syncs(self.service_config)
```

It will be formatted like this after running `make autoformat`.

Reply:
By inlined, I meant [method inlining](https://refactoring.guru/inline-method). `load_max_concurrent_access_control_syncs` just gets a key from config and IMO it does not make a sense to have it a separate method